webui/react/src/components/UPlot/UPlotScatter/UPlotScatter.utils.ts

```diff
@@ -1,5 +1,6 @@
 import uPlot from 'uplot';
 
+import { Scale } from 'components/ScaleSelectFilter';
```

Comment:
**chore question:** would you mind adding a test file for `UPlotScatter.utils.ts` 🙏